src/remoteConnector.ts

```diff
@@ -806,8 +807,22 @@ export default class RemoteConnector extends Disposable {
 				this.telemetry.sendRawTelemetryEvent('vscode_desktop_ssh', { kind: 'gateway', status: 'failed', reason: e.toString(), ...params, gitpodVersion: gitpodVersion.raw, userOverride });
 				if (e instanceof NoSSHGatewayError) {
 					this.logger.error('No SSH gateway:', e);
-					vscode.window.showWarningMessage(`${e.host} does not support [direct SSH access](https://github.com/gitpod-io/gitpod/blob/main/install/installer/docs/workspace-ssh-access.md), connecting via the deprecated SSH tunnel over WebSocket.`);
-					// Do nothing and continue execution
+					const ok = 'Ok';
```

Comment:
nit: change casing [[1](https://www.writing-skills.com/ok-ok-okay-how-do-you-write-ok)]
```suggestion
					const ok = 'OK';
```

Reply:
Actually @jeanp413 can we define this variable a level up so that we don't have to initialize the same variable on line 819?

Reply:
I don't want to pollute other scopes as the actions are different depending on message, if this increases then I'll create another file with actions constants :thinking: